model/sensor.go

```diff
@@ -48,14 +48,54 @@ const (
 	nArgsQuadratic = 3
 )
 
+// Func defines the function used to calculate the value of the sensor.
+type Func string
+
 // The keys for the function types.
 // See also SensorFuncs below.
 const (
-	keyNone      = "none"
-	keyScale     = "scale"
-	keyLinear    = "linear"
-	keyQuadratic = "quadratic"
-	keyCustom    = "custom"
+	keyNone      Func = "none"
```

Comment:
Do we need to prefix with ```key``` ?

Reply:
I was just using whatever was there, happy to change